ludwig/schema/features/binary_feature.py

```diff
@@ -24,6 +25,7 @@
 
 
 @DeveloperAPI
+@gbm_defaults_config_registry.register(BINARY)
```

Comment:
From the class name (`BinaryInputFeatureConfigMixin`) alone it was not clear to me why we wouldn't also register this class to the `ecd_defaults_config_registry`. However, upon closer inspection I see that `@ecd_defaults_config_registry.register(BINARY)` decorates the `BinaryDefaultsConfig` class.

Could you add:
- a docstring on `gbm_defaults_config_registry` (in `ludwig/schema/features/utils.py`) that the only parameters that can be specified for the `defaults` section for GBM feature types are `preprocessing` parameters
- a docstring on  `ecd_defaults_config_registry` (in `ludwig/schema/features/utils.py`) explaining the pattern of marshmallow configs that it registers?